ibmcloud/README.md

```diff
@@ -160,6 +160,60 @@ You need to build a pod VM image for peer pod VMs. A pod VM image contains the f
 
 The build scripts are located in [ibmcloud/image](./image). The prerequisite software to build a pod VM image is already installed in the worker node by [the Ansible playbook](terraform/cluster/ansible/playbook.yml) for convenience.
 
+You need to configure Cloud Object Storage (COS) to upload your custom VM image.
+
+https://cloud.ibm.com/objectstorage/
+
+First, create a COS service instance if you have not create one. Then, create a COS bucket with the COS instance. The COS service instance and bucket names are necessary to upload a custom VM image.
+
+Next, you need to grant access to COS to import images as described at [https://cloud.ibm.com/docs/vpc?topic=vpc-object-storage-prereq&interface=cli](https://cloud.ibm.com/docs/vpc?topic=vpc-object-storage-prereq&interface=cli).
+
+ibmcloud login -r jp-tok -apikey $api_key
+COS_INSTANCE_GUID=$(ibmcloud resource service-instance --output json "$IBMCLOUD_COS_SERVICE_INSTANCE" | jq -r '.[].guid')
+ibmcloud iam authorization-policy-create is cloud-object-storage Reader --source-resource-type image --target-service-instance-id $COS_INSTANCE_GUID
+
+You can use a Terraform template located at [ibmcloud/terraform/podvm-build](./terraform/podvm-build) to use Terraform and Ansible to build a pod VM image, upload it to a COS bucket and verify it.
+
+Create the `terraform.tfvars` in [the template directory](./terraform/podvm-build). `terraform.tfvars` should look like this.
+
+```
+ibmcloud_api_key = "<your API key>"
+ibmcloud_user_id = "<IBM Cloud User ID>"
+cluster_name = "<cluster name>"
+cos_service_instance_name = "<COS Service Instance Name>"
+cos_bucket_name = "<COS Bucket Name>"
+```
+
+If you used the Terraform templates in [common](./terraform/common) and [cluster](./terraform/cluster) to create the VPC and VSIs, you should set `ibmcloud_api_key` and `cluster_name` to the same values as those you entered in `terraform.tfvars` for those templates.
+
+`ibmcloud_user_id` should be set to the IBM Cloud user ID who owns the API key `ibmcloud_api_key`. You can look up the user ID using.
+
+```
+ibmcloud account users
+```
+
+If `ibmcloud account users` displays multiple user IDs, choose the user ID whose state is `ACTIVE`.
+
+The `cos_service_instance_name` and `cos_bucket_name` tfvars are optional and default to values based on your `cluster_name` tfvar if not set, as per the example in the following table.
+
+| Resource | Value |
+|----------|-------|
+| `cluster_name` tfvar | peer-pod-cluster |
+| COS Service Instance name | peer-pod-cluster-cos-service-instance | 
+| COS Bucket name | peer-pod-cluster-cos-bucket |
+
+For uploding the pod VM image using Terraform, the COS Bucket must be a regional bucket in the same region (default `jp-tok`) as the VPC and VSIs.
+
```

Comment:
Better move L262 to here.

**Note:** The `Operator` and `Console Admin` roles must be [assigned](https://cloud.ibm.com/docs/vpc?topic=vpc-vsi_is_connecting_console&interface=ui) to the user.

Reply:
Thanks. I moved that line to line 209.

Do you think we should keep a copy of that line on L262 in case the user isn't using Terraform and only reads the instructions for building the pod VM image without Terraform?

Reply:
I would like remove all no-terraform instructions for build pod VM image which happened on worker node.

We can only keep terraform instruction to pod VM image but we should add a `Note:` to say that the s390x based pod VM image only can be built from s390x based client machine.

Reply:
I removed the no-Terraform instructions.